src/ReportGenerator.Core/Reporting/Builders/Rendering/HtmlRenderer.cs

```diff
@@ -626,7 +626,8 @@ public void KeyValueRow(string key, string value)
         /// <param name="files">The files.</param>
         public void KeyValueRow(string key, IEnumerable<string> files)
         {
-            string value = string.Join("<br />", files.Select(v => string.Format(CultureInfo.InvariantCulture, "<a href=\"#{0}\" class=\"navigatetohash\">{1}</a>", WebUtility.HtmlEncode(StringHelper.ReplaceNonLetterChars(v)), WebUtility.HtmlEncode(v))));
+            int fileIndex = 0;
+            string value = string.Join("<br />", files.Select(v => string.Format(CultureInfo.InvariantCulture, "<a href=\"#{0}\" class=\"navigatetohash\">{1}</a>", WebUtility.HtmlEncode(StringHelper.ReplaceNonLetterChars(v)), WebUtility.HtmlEncode($"File {fileIndex++}: " + v))));
```

Comment:
Assign a file index to each file to shown on the class coverage page. So that we can reference to this file index in those method instances - when there are multiple instances of the same method listed on that page.